mobile/src/lib/cozy-helper.js

```diff
@@ -34,3 +34,16 @@ export const initBar = () => {
     lang: 'en'
   })
 }
+
+export const isClientRegistered = async (client) => {
+  return await cozy.client.auth.getClient(client).then(client => true).catch(() => false)
```

Comment:
In fact, here is the failing request on `getClient`:

```
curl 'http://cozy.local:8080/auth/register/977aa3b0d9e520699f9f9650e000d64a' -H 'authorization: Bearer eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJyZWdpc3RyYXRpb24iLCJpYXQiOjE0ODg4MTE5NzIsImlzcyI6ImNvenkubG9jYWw6ODA4MCIsInN1YiI6Ijk3N2FhM2IwZDllNTIwNjk5ZjlmOTY1MGUwMDBkNjRhIn0.Mtk5qDQlDrrdbyawE9LYvn3UOOIVuJEgvh1k9EZSsHZhpcETRYxNIEBDcEYPeJm6rXx0ZOnptSoGY7LsY6yqWA' -H 'accept: application/json'

HTTP/1.1 404 Not Found
Content-Security-Policy: default-src 'self';img-src *;
Content-Type: application/json; charset=UTF-8
Strict-Transport-Security: max-age=31536000; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
Date: Mon, 06 Mar 2017 15:15:43 GMT
Content-Length: 28
{"error":"Client not found"}
```

And the error in the catch block is:
```
Failed to load resource: the server responded with a status of 404 (Not Found)
```